js/main.js

```diff
@@ -30,13 +26,33 @@ function tree(needle){
         output += space.repeat(x) + "*";
         for (var i = 0; i < needle.size; i++) {
           output += '\n' + space.repeat(x-i) + needle.unit.repeat(i) 
-          + needle.unit + needle.unit.repeat(i);
-          
+                    + needle.unit + needle.unit.repeat(i); 
         }
+        output += '\n' + space.repeat(x-1) + "|◙|" 
+                  + space.repeat(x) + "...and that's how the Grinch saved Christmas.";
+               
+
         console.log(output);
       }
   }
 
+//---keydown return
+
+height.addEventListener('keydown', function(event){
+  console.log(event);
+  if(event.which == 13);
```

Comment:
Make sure when you're making comparisons you are using `===`. It's a stricter way of error checking.